### DISCUSSION TITLE
HistoryGraph: unteren Balken entfernen

### DISCUSSION THREAD

**Discussion by AFPass** [Zefau/ioBroker.jarvis#1158] [Q&A] [answered]
Wie bekomme ich den Balken weg?
![grafik](https://user-images.githubusercontent.com/13101074/140069316-60fef5e3-9a6a-4b7b-990c-be51a19dab4b.png)

**Comment by Zefau** [comment, marked as answer]
Aktuell nicht. Mach gerne einen Feature Request auf

**Reply by Zefau** [reply]
@AFPass ich hab mit `v3.0.0-alpha.116` eine neue Option hinzugefügt, mit der du die Leiste ausblenden kannst:

![image](https://user-images.githubusercontent.com/14368811/140609257-5db6a059-345d-47a6-9a88-0cc92267139b.png)

**Reply by AFPass** [reply]
@Zefau passt. Danke
